### PR TITLE
style: Add small separation in scan barcode button

### PR DESCRIPTION
**Pull request by duhow**
Adds a slight separation between buttons. Can be increased to `mt-2` but this should work for mobile devices.

image
image

### CODE REVIEW

File: src/views/AddPriceSingle.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
               </v-item-group>
             </h3>
             <v-sheet v-if="productMode === 'barcode'">
-              <v-btn class="mb-2" size="small" prepend-icon="mdi-barcode-scan" @click="showBarcodeScanner">Scan a barcode</v-btn>
+              <v-btn class="mb-2 mt-1" size="small" prepend-icon="mdi-barcode-scan" @click="showBarcodeScanner">Scan a barcode</v-btn>
```
Comment (raphodn) [inline src/views/AddPriceSingle.vue:28]: mt = margin-top ([docs](https://vuetifyjs.com/en/styles/spacing/#how-it-works))
did you mean `mr` ?

Comment (Valimp) [reply src/views/AddPriceSingle.vue:28]: has this been fixed?

**Comment by duhow** [comment]
Nope, feel free to close or push any needed changes.

File: src/views/AddPriceSingle.vue (hunk 1 above)
Comment (Valimp) [reply src/views/AddPriceSingle.vue:28]: @raphodn I think @duhow wants to add margin top to make space between "Barcode" / "Category" buttons and "Scan a Barcode button"

**Pull request closed without merging**